native/cocos/renderer/gfx-gles3/GLES3Wrangler.cpp

```diff
@@ -129,7 +130,12 @@ bool gles3wClose() {
 void *gles3wLoad(const char *proc) {
     void *res = nullptr;
     if (eglGetProcAddress) res = reinterpret_cast<void *>(eglGetProcAddress(proc));
-    if (!res) res = dlsym(libegl, proc);
+    auto sdkVersion = cc::BasePlatform::getPlatform()->getSdkVersion();
+    if (sdkVersion <= 23) {
```

Comment:
On devices running Android 6.0 (API level 23) and below, libGLESv3 includes the libEGL interfaces. This is because earlier versions of Android merged the implementation of OpenGL ES, with libGLESv3 not only providing OpenGL ES 3.x functionality but also integrating EGL features.